lib/sequent/support/database.rb

```diff
@@ -64,6 +64,8 @@ def self.establish_connection(db_config, db_config_overrides = {})
           db_config = db_config.merge(db_config_overrides)
           ActiveRecord::Base.establish_connection(db_config)
         end
+
+        ActiveRecord::Base.connection.reconnect!
```

Comment:
`#debrief` Couldn't find the exact PR on Rails that changed this, but looks like `establish_connection` now may be "lazier" than before, and not actually set an `active?` connection...